skyvern/webeye/persistent_sessions_manager.py

```diff
@@ -164,7 +165,22 @@ async def close_session(self, organization_id: str, session_id: str) -> None:
                 session_id=session_id,
             )
             self._browser_sessions.pop(session_id, None)
-            await browser_session.browser_state.close()
+
+            try:
+                await browser_session.browser_state.close()
+            except TargetClosedError:
+                LOG.info(
+                    "Browser context already closed",
+                    organization_id=organization_id,
+                    session_id=session_id,
+                )
+            except Exception as e:
+                LOG.warning(
+                    "Error while closing browser session",
+                    organization_id=organization_id,
+                    session_id=session_id,
+                    error=str(e),
```

Comment:
we can remove `error=str(e),` and add this:
```suggestion
                    exc_info=True,
```